resources/en/view/Quiz_Input.view.bxb

```diff
@@ -0,0 +1,52 @@
+input-view {
```

Comment:
We usually keep the views in base (and make sure it is using template macros for dialogs that are in target language folders). This makes it easier to add new device and locale targets.